### PR TITLE
Add rationale for forbidding a buffer to be bound to transformfeedbac…

### PR DESCRIPTION
**Pull request by zhenyao**
…k and another binding point.

### CODE REVIEW

File: specs/latest/2.0/index.html [hunk 1]
```diff
@@ -2637,9 +2643,7 @@ <h2 class="no-toc">Table of contents</h2>
     </p>
 
     <div class="note rationale">
-        This restriction has been added to prevent writing to index buffers on the GPU, which would make
-        doing any CPU-side checks on index data prohibitively expensive. Handling index buffers as different
-        from other buffer data also maps better to the Direct3D API.
+        This restriction has been added to prevent undefined values when reading from or writing to a buffer object that is simultanenously bound to a transform feedback buffer binding point and elsewhere in the GL. An example of such bindings is <code>ReadPixels</code> to a pixel buffer object binding point.
```
Comment (kenrussell) [inline specs/latest/2.0/index.html]: Could you please list the section in the ES 3.0.3 spec that results in undefined behavior here? We dug it up this morning.

Also, "an example of such bindings" is a little confusing -- did you mean "an example of such a situation"?

Finally, could you please update the last modified date of the spec?

**Comment by kenrussell** [comment]
Thanks for taking care of this. Couple of minor comments.

**Comment by zhenyao** [comment]
Updated, please take another look.

**Comment by kenrussell** [comment]
Looks good to me. If no comments from @grorg or @jdashg by tomorrow, let's merge it then.

**Comment by kenrussell** [comment]
Merging now.

**Pull request merged**